share/dictionary/dhcpv4/dictionary.rfc2242

```diff
@@ -0,0 +1,14 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+##############################################################################
+#
+#  RFC2242 - NetWare/IP Domain Name and Information
+#
+#  $Id$
+#
+##############################################################################
+
+ATTRIBUTE	Netware-Domain-Name			62	octets
+ATTRIBUTE	Netware-Sub-Options			63	octets
```

Comment:
These are TLVs.  See section 3 of the RFC.

Reply:
Looks like Netware-Domain-Name is a string; Netware-Sub-Options is the TLV. Those changes are made.